vscode/src/chat/initialContext.ts

```diff
@@ -260,18 +260,18 @@ export function getCorpusContextItemsForEditorState(): Observable<
                                 authStatus
                             )
                         ),
-                        title: 'Current Remote Repository',
+                        title: 'Search current repository',
```

Comment:
nit suggestion: keep "Current" as first word
```suggestion
                        title: 'Current repository search',
```